Source/Core/Matrix2.js

```diff
@@ -409,18 +389,14 @@ define([
      */
     Matrix2.setColumn = function(matrix, index, cartesian, result) {
         //>>includeStart('debug', pragmas.debug);
-        if (!defined(matrix)) {
-            throw new DeveloperError('matrix is required');
-        }
-        if (!defined(cartesian)) {
-            throw new DeveloperError('cartesian is required');
-        }
-        if (typeof index !== 'number' || index < 0 || index > 1) {
-            throw new DeveloperError('index must be 0 or 1.');
-        }
-        if (!defined(result)) {
-            throw new DeveloperError('result is required');
-        }
+        Check.typeOf.object(matrix, 'matrix');
+        Check.typeOf.object(cartesian, 'cartesian');
+
+        Check.typeOf.number(index, 'index');
+        Check.numeric.minimum(index, 0);
+        Check.numeric.maximum(index, 1);
```

Comment:
Since I've seen this comment a few times today, I wanted to clarify by saying the rule is that parameters should be validated in the same order they are listed in the function definition.  So func(a, b, c) should validate a then b then c. I just looked and I don't think this is stated in the coding guide, I'll open a PR to add it.